src/Groups/GAPGroups.jl

```diff
@@ -113,12 +113,8 @@ check_parent(G::PermGroup, g::PermGroupElem) = (degree(G) == degree(parent(g)))
 
 
 function elements(G::T) where T <: GAPGroup
-  els = Vector{GapObj}(GAP.Globals.Elements(G.X)::GapObj)
```

Comment:
Thanks @syxp for your patch, as discussed on Slack.

FYI, even faster (by a factor 3 on my system in Julia 1.8.3) is to keep the old code and just change the first line to `els = GAP.Globals.Elements(G.X)::GapObj`.

That said, `elements` is actually undocumented; it was inspired by the GAP side of things, but later it was decided that this should just be handled by `collect`. That said, the new method is faster than `collect`, so I'll install it as a method for that. PR will come shortly